Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -272,8 +273,7 @@ def sanity_tests(self) -> Optional[CollectionResult]:
 
     @property
     def _always_installed_packs(self) -> Optional[CollectionResult]:
-        always_installed_packs_list = ALWAYS_INSTALLED_PACKS_MARKETPLACE_V2 if \
-            self.marketplace == MarketplaceVersions.MarketplaceV2 else ALWAYS_INSTALLED_PACKS_XSOAR
+        always_installed_packs_list = ALWAYS_INSTALLED_PACKS_MAPPING.get(self.marketplace, ALWAYS_INSTALLED_PACKS_XSOAR)
```

Comment:
in this case, better not use a safe getter

```suggestion
        always_installed_packs_list = ALWAYS_INSTALLED_PACKS_MAPPING[self.marketplace]
```